Formula/vlmcsd.rb

```diff
@@ -0,0 +1,28 @@
+class Vlmcsd < Formula
+  desc "KMS Emulator in C"
+  homepage "https://github.com/Wind4/vlmcsd"
+  url "https://github.com/Wind4/vlmcsd/archive/svn1113.tar.gz"
+  version "svn1113"
+  sha256 "62f55c48f5de1249c2348ab6b96dabbe7e38899230954b0c8774efb01d9c42cc"
+  head "https://github.com/Wind4/vlmcsd.git"
+
+  depends_on "gcc" => :build
+  depends_on "make" => :build
+
+  def install
+    system "make", "CC=gcc"
+    bin.install "bin/vlmcsd"
+    bin.install "bin/vlmcs"
+    man1.install "man/vlmcs.1"
+    man7.install "man/vlmcsd.7"
+    man8.install "man/vlmcsd.8"
+    man5.install "man/vlmcsd.ini.5"
```

Comment:
Is there no `make install`?

Reply:
No, there is no `make install`

Reply:
Can you make a pull request upstream to add one then? This is a pretty finicky way to install everything.

Reply:
I have tried to contact the upstream author, but I have not received a response yet.

Reply:
It may be that the upstream author does not like to use `make install`
The `deb` he provided was also installed this way
![Screenshot_2020-05-01 Wind4 vlmcsd-debian](https://user-images.githubusercontent.com/26341224/80811829-0c56f680-8bc7-11ea-8b42-7dc2ab0fbbb0.png)